content/hardware/04.pro/boards/portenta-x8/tutorials/13.wordpress-webserver/content.md

```diff
@@ -108,11 +136,28 @@ Now you should see a webpage, like the following image, in your browser.
 
 You are now free to go through the WordPress setup process and configure it however you like.
 
+### Removing the containers ###
+
+If you want to remove the container, you have to go to ```/home/fio/wordpress-test``` directory (where we previously executed the docker-compose command) and execute the following commands according to your needs:
+
+Remove the container but preserves your WordPress database:
+
+```
+docker compose down
+```
+
+Remove the container and the database:
+
+```
+docker compose down --volumes
+```
+
+To make sure that it was successful, run ```docker ps -a``` and check that the WordPress and MariaDB containers have disappeared.
+
 ## Conclusion
 
 In this tutorial, we went through how to install and run a WordPress and database container on the Portenta X8. We then accessed the WordPress site on our X8 through our web browser. So now you can set up your own WordPress site on your X8 device and access it from another device.
 
-
 ## Troubleshooting
```

Comment:
In this Troubleshooting section, the third bullet point's command is giving an error in argument and we could use instead `ip -h address`. What do you think about this?